### ISSUE TITLE
Change in server definition "docker image" not updating.

### ISSUE THREAD

**Issue by dualmacops** [PufferPanel/PufferPanel#1038]
I changed the Enviroment variable "Docker image" from openjdk:8-alpine to openjdk:8-slim, pressed save and then run.

The server ran in with the openjdk:8-alpine image.

The issue was solved by stoping the server and pressing the "reload server data from disk".

**Comment by LordRalex** [comment]
Anything changing in the environment is currently designed to not be changed while it's running, which with docker.... does not make as much sense. This is because it's specific to the server running, and changing it on the fly is a bit dangerous. It'd be bad for example if the "root" changed while the server is doing stuff. That said, we could have it so that if the server is "offline", it applies such changes immediately, or when it goes back offline.

**Comment by dualmacops** [comment]
> Anything changing in the environment is currently designed to not be changed while it's running, which with docker.... does not make as much sense. This is because it's specific to the server running, and changing it on the fly is a bit dangerous. It'd be bad for example if the "root" changed while the server is doing stuff. That said, we could have it so that if the server is "offline", it applies such changes immediately, or when it goes back offline.

Maybe you implied that i changed the variable when it was runnning.
The server was STOP when changing the variable. 
Then i stoped the server again before presing the reload server data from disk

**Comment by LordRalex** [comment]
I did not imply that. I just mean that it doesn't work because we don't track the status, and so can't safely apply such changes. It isn't anything you did, it's just how it works.

**Comment by dualmacops** [comment]
The server was not running when i changed the image, I expected to boot with the new image when i pressed start, that was not the case.
I needed to press reload from diks for the changes to apply.

As nepcore said:
![image](https://user-images.githubusercontent.com/9700375/125109677-f48cd380-e0e3-11eb-847b-7ba93ec58c29.png)

**Comment by LordRalex** [comment]
Yeah, that's fine. It's just how I designed it. It won't reload that data unless you actually do use that button. Stuff in the "environment" bit can't be changed on the fly like that. You have to use the reload button.

**Comment by dualmacops** [comment]
Oh, if it's by design... it would be nice if it was some kind of warning stating that you need to press that button for the setting to apply

**Comment by nepcore** [comment]
Such a warning doesn't currently exist as I was not aware of that, that's something I'll take care of whenever I get around to it
